redisinsight/ui/src/constants/pages.ts

```diff
@@ -55,7 +57,7 @@ export const Pages = {
   triggeredFunctionsFunctions: (instanceId: string) =>
     `/${instanceId}/${PageNames.triggeredFunctions}/${PageNames.triggeredFunctionsFunctions}`,
   // rdi pages
-  rdi: '/integrate',
+  rdi,
   rdiPipeline: (rdiInstance: string) => `${rdi}/${rdiInstance}/pipeline`,
```

Comment:
Since you added `rdiPipelineManagement = 'pipeline',` does it make sense to refactor lines like this to be ``${rdi}/${rdiInstance}/${rdiPipelineManagement}``

Reply:
make sense, thanks